### ISSUE TITLE
use `logger.debug`  in `getAllToggles`

### ISSUE THREAD

**Issue by sashaaKr** [Unleash/unleash-proxy#123]
### Describe the feature request

Hey there, 

Inside the `getAllToggles` api there is a logger usage:
```ts
this.logger.info(
            'Get all feature toggles for provided context',
            inContext,
        );
```

This log can be very spammy with high traffic, and it seems the purpose of this log is for debugging/troubleshooting, so I would like to suggest changing it to:
```ts
this.logger.debug(
            'Get all feature toggles for provided context',
            inContext,
        );
```

If it is accepted as a request for change I can submit a PR for it.

**Comment by ivarconr** [comment]
yes, this was an oversight.

**Comment by ivarconr** [comment]
released as part of `0.15.1`.

**Comment by sashaaKr** [comment]
@ivarconr thank you so much for responding so quickly!
